steal.lic

```diff
@@ -74,9 +86,24 @@ class Steal
 
     start_script('performance', ['noclean']) unless settings.hide_to_steal
     npcs.each { |npc| steal_from_npc(npc, settings.npc_stealing_attempt_count) }
-    targets.each { |target| steal(target) }
+    
+    # make semi-intelligent choices about which order to steal things in:
+    # steal within one area before moving to next
+    @stealing_towns.each do |current_town|
+      targets.select { |target| target['town'] == current_town}
+      .each { |target| steal(target) }
+    end
+
+
+    # close stealing container before getting re-dressed 
+    # in case you stole something with same name as your equipment
+    fput ("close my #{@stealing_bag}") if @bin_stolen
```

Comment:
💡 Possible candidate for `DRCI.close_container?(@stealing_bag)`?  Otherwise, my next ~suggestion~ preference is remove the space between `fput` and the open paren 😁

Reply:
Not needed imo.  That's useful when you want to robust handling of close / failure to close (/ open / failure to open) but here, there's not a good code path if there is a problem AFAIK.